### ISSUE TITLE
Missing error handling

### ISSUE THREAD

**Issue by FelixRilling** [FelixRilling/lisa-bot#13]
![1583365837721blob](https://user-images.githubusercontent.com/3207763/76152917-72426880-60c5-11ea-803f-d4377a4af817.jpg)

**Comment by FelixRilling** [comment]
Error seems to happen when updating the presence with `Cannot set property 'onmessage' of null`

**Comment by FelixRilling** [comment]
Crash that could be related:
```
Mar 11 19:35:49 rilling.dev lisa-bot.sh[882]: lisa-bot/node_modules/discord.js/src/rest/RequestHandler.js:91
Mar 11 19:35:49 rilling.dev lisa-bot.sh[882]:         throw new HTTPError(error.message, error.constructor.name, error.status, request.method, request.path);
Mar 11 19:35:49 rilling.dev lisa-bot.sh[882]:               ^
Mar 11 19:35:49 rilling.dev lisa-bot.sh[882]: HTTPError [AbortError]: The user aborted a request.
Mar 11 19:35:49 rilling.dev lisa-bot.sh[882]:     at RequestHandler.execute (lisa-bot/node_modules/discord.js/src/rest/RequestHandler.js:91:15)
Mar 11 19:35:49 rilling.dev lisa-bot.sh[882]:     at processTicksAndRejections (node:internal/process/task_queues:94:5)
Mar 11 19:35:49 rilling.dev lisa-bot.sh[882]:     at async RequestHandler.push (lisa-bot/node_modules/discord.js/src/rest/RequestHandler.js:37:14)
Mar 11 19:35:49 rilling.dev lisa-bot.sh[882]:     at async CommandDispatcher.handleMessage (lisa-bot/node_modules/discord.js-commando/src/dispatcher.js:144:19)
Mar 11 19:35:49 rilling.dev lisa-bot.sh[882]: Emitted 'error' event on CommandoClient instance at:
Mar 11 19:35:49 rilling.dev lisa-bot.sh[882]:     at msgErr (node_modules/discord.js-commando/src/client.js:63:32)
Mar 11 19:35:49 rilling.dev lisa-bot.sh[882]:     at processTicksAndRejections (node:internal/process/task_queues:94:5) {
Mar 11 19:35:49 rilling.dev lisa-bot.sh[882]:   code: 500,
Mar 11 19:35:49 rilling.dev lisa-bot.sh[882]:   method: 'post',
Mar 11 19:35:49 rilling.dev lisa-bot.sh[882]:   path: '/channels/713619721425125449/messages'
Mar 11 19:35:49 rilling.dev lisa-bot.sh[882]: }
```

**Comment by FelixRilling** [comment]
See https://github.com/discordjs/discord.js/issues/3471 for details

**Comment by FelixRilling** [comment]
Attempted fix: 339ff11746570a7383d2ec725b2a13f716ed2855

**Comment by FelixRilling** [comment]
Will re-open if issue appears again
